packages/angular/build/src/utils/bundle-calculator_spec.ts

```diff
@@ -338,5 +336,38 @@ describe('bundle-calculator', () => {
         message: jasmine.stringMatching('foo.ext exceeded maximum budget.'),
       });
     });
+
+    it('yields exceeded individual file budget - 29040', () => {
```

Comment:
This is not correct, as it does not yield exceeded
```suggestion
    it('does not exceed the individual file budget limit', () => {
```